ci/setup_env.sh

```diff
@@ -0,0 +1,120 @@
+#!/bin/bash
+set -eEo pipefail
+
+export IS_A_RELEASE="false"
+export PROD_RELEASE="false"
+export DEV_RELEASE="false"
+export TEST_RELEASE="false"
+export COMMON_FILE_LOCATION='ci/common.sh'
+
+workdir="$( cd "$( dirname "${BASH_SOURCE[0]}" )/.." &> /dev/null && pwd )"
+github_tag="${GITHUB_REF_NAME:-}"
+release_branch="${RELEASE_BRANCH:-release}"
+prod_release_regex='^[0-9]+\.[0-9]+\.[0-9]+$'
+dev_release_regex='^[0-9]+\.[0-9]+\.[0-9]+(-rc[0-9]+){1}$'
```

Comment:
In previous PR it seems to me this was fixed with:
`dev_release_regex='^[0-9]+\.[0-9]+\.[0-9]+(-rc\.[0-9]+){1}$'`

Reply:
:thinking: from out last conversation with @lander86 it seems like the decision was made towards removing `.` after rc :shrug:

Reply:
Ok then it's fine to me.
Thanks